stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -321,16 +321,20 @@ public class ChatClient internal constructor(
      * Moreover, it warms up the connection, and sets up notifications.
      *
      */
-    private fun setUserWithoutConnectingIfNeeded() {
+    @InternalStreamChatApi
+    public fun setUserWithoutConnectingIfNeeded(): Boolean {
```

Comment:
The method with such a name shouldn't return anything. It's hard to treat output value. Do we really need it?

Reply:
Maybe it will be clearer if we can have something like:
```kotlin 
fun containsStoredCredentials(): Boolean {
  return  encryptedPushNotificationsConfigStore.get() != null
 }
 ```